public/js/employeeDetails.js

```diff
@@ -0,0 +1,84 @@
+const links = document.querySelectorAll('.tablink');
+const singleBody = document.querySelectorAll('.singleBody');
+
+const sendData = (form) => {
+  const formData = new FormData(form);
+  const holdData = {};
+  formData.forEach((value, key) => {
+    holdData[key] = value;
+  });
+  fetch('/employee', {
+    method: 'PUT',
+    body: JSON.stringify(holdData),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  }).then((result) => result.json())
+    .then((result) => {
+      if (result.err) {
+        return swal('', result.err, 'error');
+      }
+      return swal('', result.message, 'success');
+    }).catch(err => swal('', err, 'error'));
+};
+
+const show = (event, divName) => {
+  links.forEach((link) => {
+    link.className = link.className.replace('tabActive', '');
+  });
+  singleBody.forEach((body) => {
+    body.style.display = 'none';
+  });
+  document.getElementById(divName).style.display = 'block';
+  event.currentTarget.className += ' tabActive';
+};
+
+const editBtn = document.querySelectorAll('.editbtn');
+editBtn.forEach((x) => {
```

Comment:
use descriptive variables names for more readability
you can use `element` or `item` etc...